src/main/resources/InstructorSampleData.json

```diff
@@ -767,7 +767,11 @@
       "recipient": "alice.b.tmms@gmail.tmt",
       "responseMetaData": "A highlight for me has been putting Software Engineering skills to use.",
```

Comment:
You should remove this?

Reply:
I am not sure whether someone will or have used it to deserialize as [FeedbackResponse](https://github.com/TEAMMATES/teammates/blob/master/src/main/java/teammates/storage/entity/FeedbackResponse.java) Entity. If so, removing responseMetaData will cause error

Reply:
I don't think anyone has done (or will do) that.